seqr/management/commands/check_for_new_samples_from_pipeline.py

```diff
@@ -100,15 +107,19 @@ def handle(self, *args, **options):
         }
         updated_project_families = []
         updated_families = set()
+        split_project_pdos = {}
+        session = AirtableSession(user=None, no_auth=True)
         for project, sample_ids in samples_by_project.items():
             project_sample_data = update_sample_data_by_project[project.id]
-            notify_search_data_loaded(
+            is_internal = notify_search_data_loaded(
```

Comment:
This function is used several other places so it would make the calls everywhere else much messier. The only real alternative to what I did here is to compute `is_internal`  both outside and inside and `notify_search_data_loaded`